scripts/st2bootstrap-deb.sh

```diff
@@ -96,6 +96,30 @@ setup_args() {
   fi
 }
 
+check_st2_host() {
+  # Check that the following TCP ports are available.
+  # Abort the installation early if the required ports are being used by an existing process.
+
+  # nginx (80, 443), mongodb (27017), rabbitmq (4369, 5672, 25672), and st2 (9100-9102).
```

Comment:
Agreed - it would be helpful to list the specific ports needed in the error message. 

We (probably me) should also include something in our docs to say 'these ports are used by ST2)